abstractions/typescript/src/authentication/baseBearerTokenAuthenticationProvider.ts

```diff
@@ -1,28 +1,29 @@
 import { RequestInformation } from "../requestInformation";
-import { AuthenticationProvider } from "./authenticationProvider";
+import { AuthenticationProvider, AccessTokenProvider } from "./authenticationProvider";
 
 /** Provides a base class for implementing AuthenticationProvider for Bearer token scheme. */
-export abstract class BaseBearerTokenAuthenticationProvider implements AuthenticationProvider {
+export class BaseBearerTokenAuthenticationProvider implements AuthenticationProvider {
```

Comment:
if it's no longer abstract should it be in abstractions?

Reply:
Good catch, I think we should keep the abstract modifier.

Reply:
actually scratch that, while the class should remain in abstractions, because it's not implementing any particular authentication library, I do agree with the removal of the abstract keyword, people could want to just use the class as it while providing their own access token provider

Reply:
i agree it's no longer abstract.  I just thought this abstractions library was meant for only abstract items and this class would then just be in the core.

Reply:
resolving as this last comment seems to have answered your question.